packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -79,8 +79,11 @@ module.exports = {
       if (!validPassword) {
         return ctx.badRequest(null, ctx.request.admin ? [{ messages: [{ id: 'Auth.form.error.invalid' }] }] : 'Identifier or password invalid.');
       } else {
+        // Issue a refresh token and save to db.
+        const token = await strapi.plugins['users-permissions'].services.refreshtoken.issue(_.pick(user.toJSON ? user.toJSON() : user, ['_id', 'id']), ctx.request.header['user-agent']);
```

Comment:
You can move the function call directly in the `refreshToken` key

Reply:
done in the new commit